go/vt/tabletserver/query_executor.go

```diff
@@ -603,12 +607,12 @@ func (qre *QueryExecutor) fullFetch(conn poolConn, parsedQuery *sqlparser.Parsed
 	return qre.execSQL(conn, sql, true)
 }
 
-func (qre *QueryExecutor) fullStreamFetch(conn *DBConn, parsedQuery *sqlparser.ParsedQuery, bindVars map[string]interface{}, buildStreamComment []byte, callback func(*sqltypes.Result) error) error {
+func (qre *QueryExecutor) fullStreamFetch(conn *DBConn, parsedQuery *sqlparser.ParsedQuery, bindVars map[string]interface{}, buildStreamComment []byte, excludeFieldNames bool, callback func(*sqltypes.Result) error) error {
```

Comment:
My other functions like dbConnFetch expect a wantfields, which is non-uniform with streamFetch. But I can take care of this also in my next PR.